includes/gateways/manual.php

```diff
@@ -97,6 +97,8 @@ function give_manual_payment( $purchase_data ) {
 	$payment = give_insert_payment( $payment_data );
 
 	if ( $payment ) {
+        $is_anonymous = isset($purchase_data['post_data']['give_anonymous_donation']) && absint($purchase_data['post_data']['give_anonymous_donation']);
+        update_post_meta($payment, '_give_anonymous_donation', $is_anonymous);
```

Comment:
@JoshuaHungDinh `_give_anonymous_donation ` metadata value should be an integer when stored in a database.

ref: https://github.com/impress-org/givewp/blob/b68aab707f5c42e339a8d8327b2b6bf2ba72ba17/src/Donations/Repositories/DonationRepository.php#L348